src/routes/home/HomeScreen.tsx

```diff
@@ -40,57 +37,27 @@ const HomeScreen = ({ navigation }: HomeScreenProps): JSX.Element => {
   const { data: selectedProfessions, refresh: refreshSelectedProfessions } = useReadSelectedProfessions()
   const isCustomDisciplineEmpty = !customDisciplines || customDisciplines.length <= 0
 
-  useFocusEffect(
-    useCallback(() => {
-      refreshCustomDisciplines()
-      refreshSelectedProfessions()
-    }, [refreshCustomDisciplines, refreshSelectedProfessions])
-  )
+  useFocusEffect(refreshCustomDisciplines)
```

Comment:
why did you change these lines and removed the `useCallback`

Reply:
because I requested it :D `refreshCustomDisciplines` and `refreshSelectedProfessions` are already wrapped in `useCallback`s and therefore no additional `useCallback` is needed if called separately. I think it also makes the code a little bit easier to read and there is no need to group both refresh together.